src/content/enterprise/install/releases.mdx

```diff
@@ -5,6 +5,29 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 0.15.0
+TBD
+
+### Deprecation Notice
+- Deprecate support for Kubernetes [1.19](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.19.md)
```

Comment:
What happens on 1.19? Will it fail to run?

Reply:
1.19 is still supported internally. Still unclear whether we keep it as an officially supported version.
Related to: https://github.com/okteto/docs/pull/163#discussion_r939526036

Reply:
Ah I see. if we are putting a depreciation notice, we should include the timeline. If we are testing for 1.19 on this release, I'd say we leave the deprecation notice, and mention that we will stop supporting 0.19 in 0.16? cc @jmacelroy